src/MigrateCommand.php

```diff
@@ -334,6 +335,80 @@ private function updatePackages(array $packages) : void
         }
     }
 
+    private function updateConfigPipeline() : void
+    {
+        $this->output->write('<info>Updating config pipeline...</info>');
+
+        $files = glob('config/autoload/*.php');
+
+        foreach ($files as $file) {
+            $content = file_get_contents($file);
+
+            $newContent = $content;
+            $newContent = preg_replace(
+                '/(\\\?Zend\\\Expressive\\\)?Application::ROUTING_MIDDLEWARE/',
+                "\Zend\Expressive\Router\Middleware\RouteMiddleware::class",
```

Comment:
Not sure about this replacement - documentation states that constant should become:
```['middleware' => \Zend\Expressive\Router\Middleware\RouterMiddleware::class]```
but I am not sure, as all other middlewares are piped there just by class name.

https://docs.zendframework.com/zend-expressive/v2/reference/migration-to-v2-2/#routing-and-dispatch-constants

See also example of middleware-pipeline.global.php we delivered with v1 of expressive:
https://github.com/zendframework/zend-expressive-skeleton/blob/1.0.0/config/autoload/middleware-pipeline.global.php#L47-L59

Reply:
If you use an import you could use the class name only. I guess it's easier this way.